mage_ai/server/server.py

```diff
@@ -256,7 +256,7 @@ async def main(
         print('User authentication is enabled.')
         # We need to sleep for a few seconds after creating all the tables or else there
         # may be an error trying to create users.
-        sleep(3)
+        sleep(5)
```

Comment:
Is this an intentional change?

Reply:
yeah i wanted to give it a little longer before creating the default roles and users